packages/fast-components-react-msft/src/carousel/README.md

```diff
@@ -4,4 +4,8 @@
 
 ## Style guidance
 
-Class names have been provided and applied to capture the transition of the slides, and allow animation or transformation depending on incoming slide entry position.
\ No newline at end of file
+Class names have been provided and applied to capture the transition of the slides, and allow animation or transformation depending on incoming slide entry position.
+
+## Auto-rotation
+
+Auto-rotation can be enabled by setting the `autoplay` prop to "true" and including an optional `autoplayInterval` (defaults to 6000ms). If the carousel has an auto-rotation enabled, the automatic slide rotation should stop when any element in the carousel receives keyboard focus or hover. It does not resume unless the user activates a control to enable rotation. To support this scenario, the carousel treats `autoplay` as a controlled feature of this component. A callback prop of `onCarouselHoverOrFocus` is provided and executes when the carousel receives focus or mouse hover. It is the responsibility of those implementing the control to ensure that the auto-rotation stops and starts as expected.
```

Comment:
> _"It is the responsibility of those implementing the control to ensure that the auto-rotation stops and starts as expected."_

I can understand the desire to have this as the implementor's responsibility, but the carousel experience on the dev site feels broken and not accessible because hover/focus does not stop the carousel.

Reply:
You cannot reliably mix controlled and uncontrolled behavior. The most reliable way is to ensure that they manage this. I’m happy to go into more detail, but this is somewhat a limitation of react - additionally, I don’t think we should try and mix controlled and uncontrolled behavior in a given feature.